### DISCUSSION TITLE
How would one use bazel with editable pip installs?`

### DISCUSSION THREAD

**Discussion by tenspd137** [bazel-contrib/rules_python#2199] [Q&A]
Not even sure if it is possible, may be a dumb question - but how would someone support pip install in editable mode with bazel?  I have been using the py_wheel rule, really cool - but if I needed to do an editable install, would I maintain pyproject.toml and somehow use bazel as the backend?  Or would I possibly do something with bazel to create the editable install equivalent?  Just wondering if this situationi has occured and how people handle it if so?

Thanks!

**Comment by shayanhoshyari** [comment]
You cannot consume the pyproject.toml

However, you can use [new_local_repository](https://bazel.build/rules/lib/repo/local#new_local_repository) to reference any folder in your disk and provide a custom BUILD.bazel for it.

See more info: https://bazel.build/versions/6.1.0/build/external#bazel-projects (I don't know why this doc is missing in latest bazel docs...)
